lua/lspconfig/server_configurations/glint.lua

```diff
@@ -0,0 +1,36 @@
+local util = require 'lspconfig.util'
+
+local bin_name = 'glint-language-server'
+
+-- Glint should not be installed globally.
+local path_to_node_modules = util.find_node_modules_ancestor(vim.fn.getcwd())
+
+local cmd = { path_to_node_modules .. '/node_modules/.bin/' .. bin_name }
+
+return {
+  default_config = {
+    cmd = cmd,
+    filetypes = {
+      'html.handlebars',
+      'handlebars',
+      'typescript',
+      'typescript.glimmer',
+      'javascript',
+      'javascript.glimmer',
```

Comment:
for other reviewers, javascript.glimmer and typescript.glimmer are the filetypes assigned to `*.gts` and `*.gjs` in treesitter.